<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Z100107" name="com.YouMagicPro.LKS_tests.HistoryCalendarWorkingTest" tests="1" failures="1" timestamp="29 Jan 2013 08:26:36 GMT" time="18.624" errors="0">
  <testcase name="historyCalendarWorkingTest_C15179" time="18.624" classname="com.YouMagicPro.LKS_tests.HistoryCalendarWorkingTest">
    <failure type="com.thoughtworks.selenium.SeleniumException" message="ERROR: Element //div[@id=&#039;datepicker_670&#039;]/div[9]/table/tbody/tr/td/table/tbody[3]/tr[2]/td[2]/a/span not found">
      <![CDATA[com.thoughtworks.selenium.SeleniumException: ERROR: Element //div[@id='datepicker_670']/div[9]/table/tbody/tr/td/table/tbody[3]/tr[2]/td[2]/a/span not found
	at com.thoughtworks.selenium.HttpCommandProcessor.throwAssertionFailureExceptionOrError(HttpCommandProcessor.java:112)
	at com.thoughtworks.selenium.HttpCommandProcessor.doCommand(HttpCommandProcessor.java:106)
	at com.thoughtworks.selenium.DefaultSelenium.click(DefaultSelenium.java:193)
	at com.YouMagicPro.LKS_tests.HistoryCalendarWorkingTest.historyCalendarWorkingTest_C15179(HistoryCalendarWorkingTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- historyCalendarWorkingTest_C15179 -->
</testsuite> <!-- com.YouMagicPro.LKS_tests.HistoryCalendarWorkingTest -->
